src/BaseCommand.ts

```diff
@@ -34,17 +32,25 @@ export abstract class BaseCommandClass {
     }
     // Save optional arguments
     if (options) {
-      const { isActive, name, matchMode, requireActiveGame, includeHelpCommand } = options;
+      const { isActive, name, matchMode, requireActiveGame, includeHelpCommand, allowedMessageTypes } = options;
       if (name) this.name = name;
-      if (isActive) this.isActive = isActive;
+      if (isActive !== undefined) this.isActive = isActive;
```

Comment:
if we would change the default value it would be annoying :/ I think undefined is fine for now but as you are saying might want to change approach all together to check for valid values

Reply:
Sure. Including the `strict: true` compiler flag for typescript is probably something to strive for anyway in the long run, and that will solve the problem anyway.